ncm-systemd/src/main/pan/components/systemd/schema.pan

```diff
@@ -42,6 +42,9 @@ type ${project.artifactId}_unit_virtualization = string with match(SELF,
 #    make this more finegrained, e.g. has to be existing unit; or check types
 type ${project.artifactId}_valid_unit = string;
 
+# modes are in octal
+type ${project.artifactId}_systemd_mode = string with match(SELF, '^0?1?[0-7][0-7][0-7]');
```

Comment:
Should this be made globally available via [pan/types.pan](https://github.com/quattor/template-library-core/blob/master/pan/types.pan) ?

Reply:
I've opened a PR at quattor/template-library-core#200.

Once that one is merged, I can directly start using the type here or do I need to do anything else?